### ISSUE TITLE
Incorrect YAML syntax highlighting for version numbers with multiple periods

### ISSUE THREAD

**Issue by rpatrick00** [ajaxorg/ace#4468]
Given a YAML file that looks like this:

```yaml
topology:
    Name: todo_domain
    DomainVersion: 12.2.1.4.0
```
The `DomainVersion` field value is highlighted as a numeric constant instead of a string.

I tested the following regular expression using one of the [online regex testers](https://regex101.com/) and it is not excluding numbers with multiple dots in it.

```javascript
{
    token : "constant.numeric", // float
    regex : /(\b|[+\-\.])[\d_]+(?:(?:\.[\d_]*)?(?:[eE][+\-]?[\d_]+)?)(?=[^\d-\w]|$)/
}
```

**Comment by AlexanderMisel** [comment]
I would prefer numbers with multiple dots to be seen as numeric instead of string.

**Comment by robertpatrick** [comment]
@AlexanderMisel But numbers can have, at most, only one decimal point.  While your preference may be to highlight these as a numeric, it is not technically correct to do so.  If you try to convert such a string into a number in any programming language, it will fail.

**Comment by AlexanderMisel** [comment]
You are right. If I were to write a lexer for parsing, I would try to make it as accurate as I can.

**Comment by andrewnester** [comment]
Fixed and release in Ace 1.6.1
